apps/whale-api/src/module.indexer/model/dftx/place.auction.bid.ts

```diff
@@ -26,7 +29,7 @@ export class PlaceAuctionBidIndexer extends DfTxIndexer<PlaceAuctionBid> {
       sort: `${HexEncoder.encodeHeight(block.height)}-${transaction.txn.txid}`,
       vaultId: data.vaultId,
       index: data.index,
-      from: toBuffer(data.from.stack).toString('hex'),
+      from: fromScriptHex(toBuffer(data.from.stack).toString('hex'), this.network)?.address as string,
```

Comment:
This will cause a breaking change; how about we add a field instead of replacing it?

```suggestion
      from: toBuffer(data.from.stack).toString('hex'),
      address: fromScriptHex(toBuffer(data.from.stack).toString('hex'), this.network)?.address as string
```

Reply:
The above-suggested change is an example, this should be implemented on the controller level.

Reply:
moved to controller level